src/js/tech/html5.js

```diff
@@ -200,6 +200,40 @@ class Html5 extends Tech {
     });
   }
 
+  /**
+   * Attempt to force override of native video tracks.
+   *
+   * @param {Boolean} override - If set to true native video will be overridden,
+   * otherwise native video will potentially be used.
+   */
+  overrideNativeTracks(override) {
```

Comment:
we should add either an empty method or a default implementation in tech.js for this.